pkg/odo/cli/pipelines/bootstrap.go

```diff
@@ -28,10 +29,11 @@ var (
 // BootstrapOptions encapsulates the options for the odo pipelines bootstrap
 // command.
 type BootstrapOptions struct {
-	quayUsername string
-	baseRepo     string // e.g. tekton/triggers
-	prefix       string // used to generate the environments in a shared cluster
-
+	quayUsername       string
+	baseRepo           string // e.g. tekton/triggers
+	prefix             string // used to generate the environments in a shared cluster
+	tokenFileName      string // filename to GitHub auth token
```

Comment:
I wonder if we need these comments about filenames, they're fairly obvious from the name?

Also, the tendency in Go is to flatten names, so, `tokenFilename` and `quayIOAuthFilename` instead of the second `Name`.